### ISSUE TITLE
[Relay] Implement BatchNorm "Fission N Fusion"

### ISSUE THREAD

**Issue by jroesch** [apache/tvm#2946]
I saw a poster session tonight at SysML and think it would be interesting to implement this paper as a Relay optimization.

[RESTRUCTURING BATCH NORMALIZATION TO ACCELERATE CNN TRAINING](https://www.sysml.cc/doc/2019/18.pdf).

The paper proposes splitting batch-normalization into two separate operators which can then be fused with the preceding, and proceeding convolutions.

<img width="924" alt="Screenshot 2019-04-01 23 08 30" src="https://user-images.githubusercontent.com/696509/55379831-1776b900-54d3-11e9-8312-ae8402c6921f.png">

**Comment by sandeep06011991** [comment]
Hi Jared, 
Can I join in on this ? I am a beginner looking to find tasks to better understand the TVM eco system
